### PR TITLE
Adding DocumentManager FQCN service alias

### PR DESCRIPTION
**Pull request by danizord**
Just to allow `$serviceLocator->get(DocumentManager::class);`

Ping @Ocramius @superdweebie

### CODE REVIEW

File: src/DoctrineMongoODMModule/Module.php [hunk 1]
```diff
@@ -143,6 +143,7 @@ public function getServiceConfig()
             ),
             'aliases' => array(
                 'Doctrine\ODM\Mongo\DocumentManager' => 'doctrine.documentmanager.odm_default',
+                'Doctrine\ODM\MongoDB\DocumentManager' => 'doctrine.documentmanager.odm_default',
```
Comment (Ocramius) [inline src/DoctrineMongoODMModule/Module.php]: Aliases don't really work in ORM modules :-\

Comment (danizord) [inline src/DoctrineMongoODMModule/Module.php]: Y not? I use `$serviceLocator->get(EntityManager::class);` everywhere when using ORM

Comment (Ocramius) [inline src/DoctrineMongoODMModule/Module.php]: See following link:

[magic](https://github.com/doctrine/DoctrineORMModule/blob/3ad3fe79d384969afbc1a6373beece4abca322dc/config/module.config.php#L188-L190)

**Comment by gianarb** [comment]
Merged 7358fd28994417fadd1ad98f49fd4f708ba766f2
Thanks

**Comment by danizord** [comment]
@gianarb Why? As @Ocramius said, unfortunately this will not work.
I'll send a PR with `DocumentManagerAliasCompatFactory` when I have time ;)

**Comment by gianarb** [comment]
Reset sorry :D I close this PR :)

**Pull request closed without merging**